libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/roomlist/RoomList.kt

```diff
@@ -39,14 +39,12 @@ interface RoomList {
 
     /**
      * The source of the room list data.
-     * All: all rooms except invites.
-     * Invites: only invites.
+     * All: all rooms.
      *
      * To apply some dynamic filtering on top of that, use [DynamicRoomList].
      */
     enum class Source {
-        All,
-        Invites,
+        All
```

Comment:
This is becoming a bit useless if there is only one value, but I think we could keep this enum for clarity?

Reply:
Yeah, I wanted to keep it in case we'd need separate lists in the future again for some reason.